tests/functional/lib/helpers.js

```diff
@@ -49,90 +44,68 @@ define(
-
-    function login(context) {
-      return openPage(context, url("/login"), "body")
-        .setFindTimeout(config.wc.pageLoadTimeout)
-        .getCurrentUrl()
-        .then(function(url) {
-          // is this the "normal" login flow?
-          if (url.includes("return_to")) {
-            return (
-              context.remote
-                .findByCssSelector("#login_field")
-                .click()
-                .type(config.wc.user)
-                .end()
-                .findByCssSelector("#password")
-                .click()
-                .type(config.wc.pw)
-                .end()
-                .findByCssSelector("input[type=submit]")
-                .click()
-                .end()
-                // *Sometimes* GitHub can bring up an extra verification
-                // page if it detects that our test user is requesting
-                // access too much.
-                .findByCssSelector(".oauth-review-permissions")
-                .then(
-                  function() {
-                    // In this case, there's an extra button to click to convince
-                    // GitHub we're totally not a bot. >_>
-                    return context.remote
-                      .sleep(3000)
-                      .findByCssSelector("button.btn-primary")
-                      .click()
-                      .end();
-                  },
-                  function(err) {
-                    // Otherwise, we swallow the NoSuchElement error.
-                    return new Promise(function(resolve) {
-                      if (/NoSuchElement/.test(String(err))) {
-                        resolve(true);
-                      } else {
-                        throw err;
-                      }
-                    });
-                  }
-                )
-                .end()
-                // ...Now make sure the logged-in avatar is shown so we know we're
-                // back at the home page before we end.
-                .findByCssSelector(".wc-Navbar-avatar")
-                .end()
+    /* 
+    This method makes a call to our API to check that the server is returning fixture data,
+    it will also check if there's anything wrong with the server.
+  */
+    function checkServer() {
+      return new Promise(function(resolve, reject) {
+        var request = http.get(url("/api/issues/100"), function(response) {
+          // If the headers are from non-fixture data,
+          // they will have the "etag" property.
+          if (response.headers.hasOwnProperty("etag")) {
+            reject(
+              new Error(
+                `
+              =======================================================
+              It seems like you didn't start the server in test mode.
+              Open another terminal and window type: 
+             \x1b[32m npm run start:test\x1b[0m
+              or
+             \x1b[32m source env/bin/activate && python run.py -t\x1b[0m
```

Comment:
Can we have this second line just mention `python run.py -t`? The first does effectively the same, but assumes you used `virtualenv` in a specific way. This second way is more general.

Reply:
Yeah, sure. Makes sense.

Reply:
Thanks!